### PR TITLE
add new env var to enable console_management

### PR DESCRIPTION
**Pull request by he2ss**

### CODE REVIEW

File: docker/docker_start.sh [hunk 1]
```diff
@@ -353,6 +353,11 @@ for BOUNCER in $(compgen -A variable | grep -i BOUNCER_KEY); do
     fi
 done
 
+if [ "$ENABLE_CONSOLE_MANAGEMENT" ] != "" ]; then
```
Comment (mmetc) [inline docker/docker_start.sh:356]: there is an extra ]

**Review by blotus — APPROVED** [review]
Merge after #2601 is merged

**Pull request merged**